2_ThreeSum.cpp

```diff
@@ -114,9 +114,9 @@ int main()
 	vector<vector<int> > result = threeSum(inp);
   	for (int i = 0; i < result.size(); ++i) 
 	{
-    	for (int j = 0; j < 3; ++j)
-      		cout << result[i][j] << " ";
-    	cout << endl;
+    		for (int j = 0; j < 3; ++j)
+      			cout << result[i][j] << " ";
+    		cout << endl;

```

Comment:
Always include unit test with your code reviews.  This helps your interviewer know that you have considered all the use cases of your function and that it performs as expected for all of them.

Reply:
Thanks a lot for the code review. I'll include unit tests and make the other changes which you've pointed in my next commit.